compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -27,6 +27,46 @@ using Granularity = luci::QuantizationGranularity;
 
 namespace
 {
+// helper function to create dummy float const node at the graph
+template<Type T>
+luci::CircleConst *createDummyConst(loco::Graph *g, luci::test::ShapeU32 shape)
```

Comment:
other methods use snake case; it would be better to follow them: `create_dummy_const`